gov2/cloudfront/CreateDistribution/CreateDistribution.go

```diff
@@ -0,0 +1,134 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[gov2.cloudfront.CreateDistribution]
```

Comment:
snippet-start tags must have matching snippet-end tags in the same file to mark a block of code that is part of a snippet.